docs/installation-guide/quickstart.rst

```diff
@@ -0,0 +1,345 @@
+
+   .. code-block:: shell
+
+    $ /srv/scality/metalk8s-2.0/bootstrap.sh
+
+2. You need to install kubectl for the CLI and check if pods are running well.
+
+   .. code-block:: shell
+
+    $ yum install —disablerepo=* —enablerepo=metalk8s-* kubectl
+    $ export KUBECONFIG=/etc/kubernetes/admin.conf
+
+    $ kubectl get node
+    NAME                   STATUS    ROLES                         AGE       VERSION
+    bootstrap              Ready     bootstrap,etcd,infra,master   17m       v1.11.7
+
+    $ kubectl get pods --all-namespaces -o wide
+    NAMESPACE     NAME                                          READY     STATUS    RESTARTS   AGE       IP             NODE                  NOMINATED NODE
+    kube-system   calico-node-zw74v                             1/1       Running   0          18m       172.21.254.7   bootstrap.novalocal   <none>
+    kube-system   coredns-6b9cb79bf4-jbtxc                      1/1       Running   0          18m       10.233.0.2     bootstrap.novalocal   <none>
+    kube-system   coredns-6b9cb79bf4-tdmz8                      1/1       Running   0          18m       10.233.0.4     bootstrap.novalocal   <none>
+    kube-system   etcd-bootstrap                                1/1       Running   0          17m       172.21.254.7   bootstrap.novalocal   <none>
+    kube-system   kube-apiserver-bootstrap                      1/1       Running   0          17m       172.21.254.7   bootstrap.novalocal   <none>
+    kube-system   kube-controller-manager-bootstrap             1/1       Running   0          17m       172.21.254.7   bootstrap.novalocal   <none>
+    kube-system   kube-proxy-mwxhf                              1/1       Running   0          18m       172.21.254.7   bootstrap.novalocal   <none>
+    kube-system   kube-scheduler-bootstrap                      1/1       Running   0          17m       172.21.254.7   bootstrap.novalocal   <none>
+    kube-system   metalk8s-ui-656f6857b-cdt5p                   1/1       Running   0          18m       10.233.0.3     bootstrap.novalocal   <none>
+    kube-system   package-repositories-bootstrap                1/1       Running   0          17m       172.21.254.7   bootstrap.novalocal   <none>
+    kube-system   registry-bootstrap                            1/1       Running   0          17m       172.21.254.7   bootstrap.novalocal   <none>
+    kube-system   salt-master-bootstrap                         2/2       Running   0          17m       172.21.254.7   bootstrap.novalocal   <none>
+
+
+Adding a master to the cluster
+------------------------------
+
+Now it's time to add more node to the cluster. First you need to secure the
+cluster by adding 2 nodes with etcd and master roles.
+Here is the procedure to add one, simply do it twice to have 3
+masters (bootstrap + 2 new master).
+
+1. Copy the ssh-key to the new node
+
+   .. code-block:: shell
+
+    $ ssh-copy-id <IP-of-the-new-master-node>
+
+2. Create a yaml config file for this new master.
```

Comment:
Minor:

```suggestion
2. Create a YAML config file for this new master.
```

Reply:
Actually, might be better to describe it with accurate terms: it's a _`Node` manifest_

Reply:
Unless we're against linking to K8s documentation, if we're using K8s terms we should link to the K8s documentation of these terms where relevant.

Reply:
We're not against linking, on the contrary, it's a good idea! There's the `intersphinx` extension already, though I'm not sure if it can be used for K8s docs.